apm-sniffer/optional-plugins/fastjson-1.2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/fastjson/define/FastjsonInstrumentation.java

```diff
@@ -33,7 +33,7 @@
 import static net.bytebuddy.matcher.ElementMatchers.named;
 
 /**
- * Fastjson A fast JSON parser/generator for Java (It's the most popular
+ * Fastjson A fast JSON parser/generator for Java (It's very popular
  * json parsing library in China from Alibaba inc).
```

Comment:
Don't keep the statement. Like we mentioned, there is no point to do marketing in another open source project.

Reply:
If you want to mention Alibaba Inc. as owner, it is fine. Other things, including widely used or not, China or global. Those are irrelative message.

Reply:
Thanks for your advice. I will change it.